Patches from freebsd.  Fixes build on userland build server

commit 691f8469ca04b6b9d2944e73637378773820b415
Author: Christoph Moench-Tegeder <cmt@FreeBSD.org>

    taskcluster-taskgraph not available, don't try to download

diff --git python/sites/build.txt python/sites/build.txt
index 1654169c4a28..0000d9279345 100644
--- thunderbird-150.0/python/sites/build.txt
+++ thunderbird-150.0/python/sites/build.txt
@@ -1,6 +1,6 @@
 requires-python:>=3.9
 pth:third_party/python/vsdownload
-pypi-optional:taskcluster-taskgraph==24.1.1:toolchain artifacts will not be bootstrapped
+# pypi-optional:taskcluster-taskgraph==24.1.1:toolchain artifacts will not be bootstrapped
 vendored:testing/web-platform/tests/tools/third_party/h2/src
 vendored:testing/web-platform/tests/tools/third_party/hpack/src
 vendored:testing/web-platform/tests/tools/third_party/html5lib
diff --git python/sites/mach.txt python/sites/mach.txt
index 498e16ebba4f..4023a11d7e9a 100644
--- thunderbird-150.0/python/sites/mach.txt
+++ thunderbird-150.0/python/sites/mach.txt
@@ -58,15 +58,15 @@ pth:xpcom/geckoprocesstypes_generator
 pth:xpcom/idl-parser
 # glean-sdk may not be installable if a wheel isn't available
 # and it has to be built from source.
-pypi-optional:glean-sdk==67.3.2:telemetry will not be collected
-pypi-optional:orjson>=3.10:json operations will be slower in various tools
+# pypi-optional:glean-sdk==67.3.2:telemetry will not be collected
+# pypi-optional:orjson>=3.10:json operations will be slower in various tools
 # Mach gracefully handles the case where `psutil` is unavailable.
 # We aren't (yet) able to pin packages in automation, so we have to
 # support down to the oldest locally-installed version (5.4.2).
-pypi-optional:psutil>=5.4.2:telemetry will be missing some data
-pypi-optional:rtoml>=0.11.0:toml operations will be slower in various tools
-pypi-optional:zstandard>=0.11.1,<=0.25.0:zstd archives will not be possible to extract
-vendored-fallback:pyyaml:third_party/python/pyyaml/lib/:faster native loading is disabled
+# pypi-optional:psutil>=5.4.2:telemetry will be missing some data
+# pypi-optional:rtoml>=0.11.0:toml operations will be slower in various tools
+# pypi-optional:zstandard>=0.11.1,<=0.25.0:zstd archives will not be possible to extract
+vendored:third_party/python/pyyaml/lib/
 vendored:third_party/python/ansicon
 vendored:third_party/python/appdirs
 vendored:third_party/python/async_timeout

